app/services/bgs/awards_service.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module BGS
+  class AwardsService < BaseService
+    def get_awards
+      @service.awards.find_award_by_participant_id(@user.participant_id)
+    rescue => e
```

Comment:
Is there a specific exception class you can rescue from - this may swallow unrelated errors unexpectedly